src/main/scala/org/apache/spark/streamdm/evaluation/BasicClassificationEvaluator.scala

```diff
@@ -38,14 +43,29 @@ class BasicClassificationEvaluator extends Evaluator{
    * @return a stream of String with the processed evaluation
    */
   override def addResult(input: DStream[(Example, Double)]): DStream[String] = {
-    //print the confusion matrix for each batch
-    val pred = ConfusionMatrix.computeMatrix(input)
-    pred.map(x => {"%.3f,%.0f,%.0f,%.0f,%.0f"
-      .format((x._1+x._4)/(x._1+x._2+x._3+x._4),x._1,x._2,x._3,x._4)})
+    val confusionMatrix = ConfusionMatrix.computeMatrix(input)
+    confusionMatrix.map(calculateMetrics)
+  }
+
+  /**
+    * Helper function to calculate several evaluation metrics based on a confusion matrix
+    * @param confMat
+    * @return
+    */
+  def calculateMetrics(confMat : Map[String,Double] ): String = {
+    val accuracy = (confMat{"tp"}+confMat{"tn"})/(confMat{"tp"}+confMat{"tn"}+confMat{"fp"}+confMat{"fn"})
```

Comment:
Format.

Reply:
Hi @zhangjiajin , 
I have updated the format in this function.